ensure-no_std/Cargo.toml

```diff
@@ -9,7 +9,7 @@ cairo-felt = { path = "../felt", default-features = false, features = [
 ] }
 cairo-vm = { path = "../vm", default-features = false }
 
-wee_alloc = "0.4.5"
+esp-alloc = "0.3.0"
```

Comment:
Is this correct? The description on crates.io seems to suggest this is ESP only.

Reply:
It seems to compile fine, so I would assume it is. As we use this for compilation testing, I just looked for a _no-std_ allocator. I also found [this one](https://crates.io/crates/alloc-no-stdlib), which seems more general.

Reply:
Hmmm OK, if it's just compilation testing and it compiles, it works for me.